src/TuDa.CIMS.Web/Components/AssetSearch.razor

```diff
@@ -0,0 +1,22 @@
+﻿@* Search bar for assets *@
+<div class="position-relative"
+     style="max-width: 600px;">
+    <input class="form-control rounded-pill"
+           id="prototypeSearchBar"
+           placeholder="Suche">
+
+    <!-- Search button -->
+    <button type="button"
+            class="position-absolute end-0 top-50 translate-middle-y me-2 btn p-0 border-0 bg-transparent">
+
+        <!-- Search icon -->
+        <i class="bi bi-search position-absolute"
+           style="top: 50%; right: 15px; transform: translateY(-50%); color: #aaa;">
+        </i>
+    </button>
+</div>
+
+@* Link to Bootstrap Icons *@
+<link
+    href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.0/font/bootstrap-icons.css"
+    rel="stylesheet"/>
```

Comment:
:thinking: Is this really necessary? I thought the icons are included in the normal package

Reply:
Ok, it actually is.

Reply:
:wrench: You can also just move this to `App.razor`, but then add a TODO comment above it.